### PR TITLE
DBZ-4705 Add bytes support for blob and binary types

### PR DESCRIPTION
**Pull request by shichao-an**
- Store raw bytes instead of string in `VitessColumnValue` and `ReplicationMessageColumn` used internally in value resolving and converters
- Add bytes support for blob and binary types
- Add support for binary.handling.mode and enable it in config

This fixes: https://issues.redhat.com/browse/DBZ-4705

cc @y5w @keweishang

### CODE REVIEW

**Review by jeffchao — APPROVED** [review]
LGTM.

Very straightforward, thanks. I'd like a second opinion from @y5w before merging.

**Comment by gunnarmorling** [comment]
Nice one! I suppose there'll be a PR for the main repo to discuss the new
type mappings in the docs?

File: src/main/java/io/debezium/connector/vitess/connection/ReplicationMessageColumn.java
Comment (shichao-an) [reply src/main/java/io/debezium/connector/vitess/connection/ReplicationMessageColumn.java:59]: If you meant `asString()` method of the `VitessColumnValue` class. Yes, the current assumption is that the VARCHAR and all other non-bytes types are UTF-8 encoding, .e.g for timestamps, numeric types, etc. I can update the javadoc

**Comment by shichao-an** [comment]
> Nice one! I suppose there'll be a PR for the main repo to discuss the new type mappings in the docs?
> […](#)
> Message ID: <debezium/debezium-connector-vitess/pull/74/review/875096699@ github.com>

Yes, will follow up with a PR to update the doc on the mapping. Thanks

File: src/main/java/io/debezium/connector/vitess/connection/ReplicationMessageColumn.java
Comment (y5w) [reply src/main/java/io/debezium/connector/vitess/connection/ReplicationMessageColumn.java:59]: nit: consider move this into a static util wrapper together with what we have on VitessColumnValue.java

Comment (shichao-an) [reply src/main/java/io/debezium/connector/vitess/connection/ReplicationMessageColumn.java:59]: Since this `toString()` here is for debugging purpose, and I feel it's unnecessary to introduce a wrapper layer for a one-line code.

**Pull request merged**
